app/views/content/life-as-a-teacher/explore-subjects/languages.md

```diff
@@ -3,6 +3,8 @@ title: Become a languages teacher
 title_paragraph: |-
   <p>
   Learning a language teaches pupils more than just grammar and vocabulary. As a languages teacher, you'll help pupils gain a greater appreciation and understanding of the world.</p>
+  <p>
+  Tax-free bursaries of $bursaries_postgraduate_languagesother$ are availabe for eligilbe trainee teachers in all languages. For French, German and Spanish trainee teachers, tax-free scholarships of $scholarships_languagesfrenchgermanspanish$ are available.</p>
```

Comment:
```suggestion
  Tax-free bursaries of $bursaries_postgraduate_languagesother$ are available for eligible trainee teachers in all languages. For eligible French, German and Spanish trainee teachers, tax-free scholarships of $scholarships_languagesfrenchgermanspanish$ are available.</p>
```

Also - would it be clearer to split into separate paras? But your call @emmaachesongray